core/src/main/java/com/linecorp/armeria/internal/common/util/ServiceNamingUtil.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright 2021 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+package com.linecorp.armeria.internal.common.util;
+
+import com.linecorp.armeria.common.RpcRequest;
+import com.linecorp.armeria.common.util.Unwrappable;
+import com.linecorp.armeria.server.HttpService;
+
+public final class ServiceNamingUtil {
+
+    private static final String GRPC_SERVICE_NAME = "com.linecorp.armeria.internal.common.grpc.GrpcLogUtil";
```

Comment:
Could we have a test case in `armeria-grpc` that ensures the class name didn't change? e.g.

```java
assertThat(GRPC_SERVICE_NAME).equals(GrpcLogUtil.class.getName());
```

Reply:
Thanks for taking long threads! Sure. I added and also updated a hard-coded part in [`AccessLogComponent`](https://github.com/line/armeria/pull/3366/files#diff-5455e420a88bdc650bf7feaee0985460d6983cde8550471f264cfec8288630d9R301).